ios-sdk/TangramVC.swift

```diff
@@ -12,6 +12,7 @@ class TangramVC:  TGMapViewController{
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        self.queueSceneUpdate("sources.mapzen.url_params.api_key", withValue: "vector-tiles-7bfyfz8")
```

Comment:
Ach, disregard my comment above! I hadn't noticed that this branch also updates the scene file. 

Another problem could be that scene updates don't create all necessary nodes along the path to a target node. In other words, it acts like `mkdir` and not `mkdir -p`. Since the version of bubble-wrap here doesn't already have a `url_params` node, you could try to make this update something like:
```
queueSceneUpdate("sources.mapzen.url_params", "{ api_key: xxxxxx }")
```